presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -276,6 +276,10 @@ Name                                        Description
                                             are emitted to ``stdout``.
 ``control.table-prefix``                    The table prefix to be appended to the control target table.
 ``test.table-prefix``                       The table prefix to be appended to the test target table.
+``control.reuse-table``                     If true, reuse the output table of the control source Insert and CreateTableAsSelect
+                                            query. Otherwise, run the control source query and write to a temporary table.
+``test.reuse-table``                        If true, reuse the output table of the test source Insert and CreateTableAsSelect
```

Comment:
```suggestion
``test.reuse-table``                        If ``true``, reuse the output table of the test source Insert and CreateTableAsSelect
```

Reply:
Rebased and this change is in previous PR.